tests/filesTest.js

```diff
@@ -629,7 +631,7 @@ describe('Main: Currently testing files management,', function () {
           headers: {
             authorization: getAuthHeaders(config.testUser, config.testUserPassword),
             Destination: MatchersV3.fromProviderState(
-              `\${providerBaseURL}${destinationWebDavPath}`,
+              `\${providerBaseURL}/${destinationWebDavPath}`,
```

Comment:
this change may not be required as it depends on `PROVIDER_BASE_URL` env variable.
you can have trailing`/` as `PROVIDER_BASE_URL=https/localhost:9200/` while running provider tests